crates/net/network/src/transactions/mod.rs

```diff
@@ -75,13 +75,16 @@ pub use validation::*;
 const PEER_TRANSACTION_CACHE_LIMIT: usize = 1024 * 10;
 
 /// Soft limit for NewPooledTransactions
-const NEW_POOLED_TRANSACTION_HASHES_SOFT_LIMIT: usize = 4096;
+const NEW_POOLED_TRANSACTIONS_BROADCAST_PACKET_HASHES_COUNT_SOFT_LIMIT: usize = 4096;
 
-/// Soft limit for the response size of a GetPooledTransactions message (2MB) in bytes. Standard
-/// maximum response size. See specs
+/// Soft limit for the message of full transactions in bytes.
+const FULL_TRANSACTIONS_BROADCAST_MSG_BYTE_SIZE_SOFT_LIMIT: usize = 100 * 1024;
+
+/// Soft limit for the response size of a [`GetPooledTransactions`] message (128 KiB) in bytes.
+/// Standard maximum response size is 2 MiB. See specs
 ///
 /// <https://github.com/ethereum/devp2p/blob/master/caps/eth.md#protocol-messages>.
-const POOLED_TRANSACTIONS_RESPONSE_SOFT_LIMIT_BYTE_SIZE: usize = 2 * 1024 * 1024;
+const POOLED_TRANSACTIONS_RPC_RESPONSE_SOFT_LIMIT_BYTE_SIZE: usize = 128 * 1024;
```

Comment:
why is this called RPC?

Reply:
ah I see the devp2p specs don't make the distinction between gossip messages and req-response messages. big difference though in terms of the effects of a too big message. for gossiped messages there is amplification of the "too big message problem" where as req-response is just effecting sender and receiver.

Reply:
in cl we refer to the messages as gossip messages and rpc messages. gossip messages since they are being sent as part of a gossip protocol. rpc messages are request-response messages. the former is multicast. the latter is unicast. hence why I am also reluctant to say "broadcast" for the mempool, since it's leaning more towards multicast actually.

Reply:
I would definitely call it something different than RPC, because we actually serve json-RPC requests, so it would be very confusing

Reply:
yepp, on it, ported it from cl